metadata.rb

```diff
@@ -0,0 +1,9 @@
+name 'yum-almalinux'
+maintainer        'Sous Chefs'
+maintainer_email  'help@sous-chefs.org'
+license           'Apache-2.0'
+description 'Installs/Configures AlmaLinux yum repositories'
+version '0.1.0'
+chef_version '>= 16.0'
```

Comment:
This is the first version which included support for identifying AlmaLinux in ohai.
```suggestion
chef_version '>= 16.10'
```